daft/daft/__init__.pyi

```diff
@@ -1642,9 +1647,8 @@ class LogicalPlanBuilder:
         left_on: list[PyExpr],
         right_on: list[PyExpr],
         join_type: JoinType,
-        strategy: JoinStrategy | None = None,
-        join_prefix: str | None = None,
-        join_suffix: str | None = None,
+        join_strategy: JoinStrategy | None = None,
+        column_renaming_params: JoinColumnRenamingParams | None = None,
```

Comment:
i'd prefer deprecating `join_prefix` and `join_suffix` instead of just flat out removing them.

Reply:
i'm kind of also on the fence of leaving these as is for the dataframe api. IMO, it's a bit cleaner to do 
```py
df.join(df2, prefix="df2.", suffix="_joined")
```
instead of 
```py
from daft.daft import JoinColumnRenamingParams

df.join(df2, column_renaming_params=JoinColumnRenamingParams(prefix="df2.", suffix="_joined"))
```

Reply:
The API for the actual dataframe operation has not changed, this is just for the builder. Do you think we should be concerned about breaking the builder API?